pkg/buses/reconciler.go

```diff
@@ -171,6 +171,7 @@ func NewReconciler(
 				if oldBus.ResourceVersion == newBus.ResourceVersion {
 					// Periodic resync will send update events for all known Buses.
 					// Two different versions of the same Bus will always have different RVs.
+					logger.Info("Identical resource version, ignoring update")
```

Comment:
If this log message is useful, we should probably also apply it to the other informer event handlers. Otherwise, delete it.